Tests/UnitTests/fss-search-results.component.spec.ts

```diff
@@ -1,27 +1,34 @@
 import { NO_ERRORS_SCHEMA } from '@angular/core';
-import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { TestBed } from '@angular/core/testing';
+import { HttpClientModule } from '@angular/common/http';
 
 import { FssSearchResultsComponent } from '../../src/app/features/fss-search/fss-search-results/fss-search-results.component';
+import { FileShareApiService } from '../../src/app/core/services/file-share-api.service';
+import { AppConfigService } from '../../src/app/core/services/app-config.service';
 
 describe('FssSearchResultsComponent', () => {
-  let component: FssSearchResultsComponent;
-  let fixture: ComponentFixture<FssSearchResultsComponent>;
+  let component: FssSearchResultsComponent;  
+  let fileShareApiService: FileShareApiService;  
 
   beforeEach(async () => {
     await TestBed.configureTestingModule({
-      declarations: [ FssSearchResultsComponent ],
+      imports: [HttpClientModule,        ],
+      declarations: [FssSearchResultsComponent],
+      providers: [FileShareApiService],
       schemas: [NO_ERRORS_SCHEMA]
     })
-    .compileComponents();
-  });
-
-  beforeEach(() => {
-    fixture = TestBed.createComponent(FssSearchResultsComponent);
-    component = fixture.componentInstance;
-    fixture.detectChanges();
+      .compileComponents();
+    AppConfigService.settings = {
+      fssConfig: {
+        "apiUrl": "https://fss-dev-webapp.azurewebsites.net/"
```

Comment:
Should we be hard coding the Uri into the code files? (sorry not up to speed at the moment so not sure where it is used)

Assume this might break if the Uri changes?

Reply:
@martyn-Federal This code is in the test case file, so it won't be executed. But I have modified the code with dummy url and pushed the changes

Reply:
Thanks Martin. 
I modified the urls like you suggested and pushed the code.